test/levels.test.js

```diff
@@ -257,12 +258,16 @@ test('resets levels from labels to numbers', async ({ is }) => {
   instance.info('hello world')
 })
 
+test('aliases changeLevelName to levelKey', async ({ is }) => {
+  is(pino({ changeLevelName: 'priority' })[levelKeySym], 'priority')
```

Comment:
If there is a preference to avoid looking at "private" properties, there are other ways this can be tested.

Reply:
yeah if possibly avoid private state in tests